administrator/language/en-GB/en-GB.com_media.ini

```diff
@@ -16,6 +16,9 @@ COM_MEDIA_CREATE_NEW_FOLDER_ERROR="Error creating folder."
 COM_MEDIA_DECREASE_GRID="Decrease grid size"
 COM_MEDIA_DELETE_ERROR="Error deleting the item."
 COM_MEDIA_DELETE_SUCCESS="Item deleted."
+COM_MEDIA_CONFIRM_DELETE_MODEL_DESC="Are you sure you want to delete this item ?"
```

Comment:
remove the space before the ?

(if this is for folders as well as for an individual item AND/OR if this can be used for multiple items then it would be better to just have
+COM_MEDIA_CONFIRM_DELETE_MODEL_DESC="Are you sure you want to delete this?"